lib/v2/screens/profile_screens/guardians/invite_guardians/invite_guardian_screen.dart

```diff
@@ -57,7 +57,7 @@ class InviteGuardians extends StatelessWidget {
                         children: state.selectedGuardians
                             .map((e) => SearchResultRow(
                                   account: e.account,
-                                  name: e.nickname ?? e.account,
+                                  name: e.nickname != "" ? e.nickname : e.account,
```

Comment:
`e.nickname.isNotEmpty ? e.nickname : e.account,`

Reply:
this should be nullable. 
e.nickname  == null

Reply:
Nickname is no longer nullable, I changed this, hence changing this line of code. But nickname can be empty in which case we want to show the account name